locality_sensitive_hashing/lsh.py

```diff
@@ -542,29 +535,25 @@ def return_results(lsh_similarity_matrix: LSHSimilarityMatrix) -> list[tuple[int
 
     Arguments:
         lsh_similarity_matrix: LSHSimilarityMatrix
-            A vector of size `L choose 2` where `L` is the number of documents. Each item is the
-            similarity of a pair of candidate documents. The vector is stored in a lower triangular
-            matrix, where the index of each item is calculated using the `get_triangle_index`
-            function. This is the return value of the `candidates_similarities` function.
+            A matrix of size `L x L` where `L` is the number of documents. Each item (i, j) is the
+            similarity of a pair of candidate documents. The matrix is the return value of the
+            `candidates_similarities` function. The matrix is stored as an upper triangular matrix.
 
     Returns:
-        list[tuple[int, int]]
-            A list of document index pairs that are above the threshold similarity.
+        list[tuple[str, str]]
+            A list of tuple of document pairs of more than `t`% similarity.
     """
     threshold = parameters_dictionary["t"]
 
     # Find the triangle indices of the similarity matrix that are above the
     # threshold
     indices_above_threshold = np.argwhere(lsh_similarity_matrix >= threshold)
```

Comment:
Is it still correct to say that we find the triangle indices here?

Reply:
no

Reply:
Fixed in 85f478d